jitbuilder/release/src/TransactionalOperations.cpp

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
```

Comment:
yes, TABORT is related to nothing here. I added it only considering the integrity of TM services (since tstart/tend/tabort always appears together) and just in case others might use it in the future. Should I remove it?